### ISSUE TITLE
Changing from an account to another does not modify the .actor element

### ISSUE THREAD

**Issue by DavidLibeau** [Chocobozzz/PeerTube#611]
* **PeerTube version or URL:** Beta.7
* **Browser name/version:** Firefox 60

* **What is the expected behaviour?**

Change infos of .actor on the accounts page.
Step to reproduce : 
- Go to an account page (that is not your account)
- Click on "My public profile" button

* **What do you see instead?**

The infos of .actor are not changing but the url, the content and the tabs are ok.

* **Browser console log if useful (Gist/Pastebin...):**
```
Rebuilt pages with 20 elements per page.
```

**Comment by Chocobozzz** [comment]
I'm sorry but I don't understand. Could you rephrase the steps to reproduce? In particular I don't understand "Change infos of .actor on the accounts page.".

**Comment by DavidLibeau** [comment]
.actor is a CSS query. But actually, I seems to affect the whole accounts page.

Here is a video : 

![peertubebug](https://user-images.githubusercontent.com/11574966/40854540-5bef26dc-65c1-11e8-82aa-4129ec5e0f0e.gif)

**Comment by Chocobozzz** [comment]
Thanks, the video was very helpful! Definitely a bug :)

**Comment by Chocobozzz** [comment]
Should be fixed: https://github.com/Chocobozzz/PeerTube/commit/734a5ceb3d04088743d72babcb9b05e6142043f6

Thanks again for the bug report @DavidLibeau
